plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -35,6 +35,28 @@ void testShowTables()
         assertUpdate("CREATE TABLE faker.default.test (id INTEGER, name VARCHAR)");
         assertTableColumnNames("faker.default.test", "id", "name");
     }
+    
+    @Test
+    void testRenameTable()
+    {
+        assertQuery("SHOW TABLES FROM faker.default LIKE 'original_table'", "SELECT '' WHERE false");
```

Comment:
This is not formatted correctly, you need to add one more space for indentation. Try setting up your IDE to format all files automatically.